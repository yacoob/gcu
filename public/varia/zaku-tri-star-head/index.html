<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <!--[if lt IE 9]>
  <script src='https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js'></script>
  <script src='https://oss.maxcdn.com/respond/1.4.2/respond.min.js'></script>
  <![endif]-->
  <link href='/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='//syn.tactical-grace.net/f/full/00847.jpg' rel='image_src'/>
  <meta property='og:image' content='https://syn.tactical-grace.net/f/full/00847.jpg'/>
  <meta name='description' content='tactical grace presents: Zaku II Head (Black Tri-Star)'/>
  <link href='http://gcu.tactical-grace.net/varia/zaku-tri-star-head/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>Varia: Zaku II Head (Black Tri-Star) | tactical grace</title>
  <meta content='Varia: Zaku II Head (Black Tri-Star) | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/varia/zaku-tri-star-head/'>Varia: Zaku II Head (Black Tri-Star)</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a href='/varia/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a href='/varia/shops-akihabara/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2016-12-26'>2016-12-26</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00835.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00835.jpg' title='Looks innocent enough.' alt='Looks innocent enough.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00836.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00836.jpg' title='The pistons at the bottom just love to fall apart.' alt='The pistons at the bottom just love to fall apart.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00837.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00837.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00838.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00838.jpg' title='None of that is visible upon completion :3' alt='None of that is visible upon completion :3' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00839.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00839.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00840.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00840.jpg' title='Welcome to nubmark hell.' alt='Welcome to nubmark hell.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00841.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00841.jpg' title='&#34;Gahh!&#34;' alt='&#34;Gahh!&#34;' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00842.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00842.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00843.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00843.jpg' title='One of the pieces that required glue - the main visor wiper.' alt='One of the pieces that required glue - the main visor wiper.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00844.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00844.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00845.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00845.jpg' title='Glue was required for all small protruding bits.' alt='Glue was required for all small protruding bits.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00846.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00846.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00847.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00847.jpg' title='Screwed up the decals a bit, had to improvise.' alt='Screwed up the decals a bit, had to improvise.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/00848.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/00848.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
</div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='http://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link type='text/css' rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/lightgallery/1.3.6/css/lightgallery.min.css' />
<!-- include lightbox js/css here -->
<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js'></script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/lightgallery/1.3.6/js/lightgallery.min.js'></script>
<script src='/js/lg-fullscreen.min.js'></script>
<script src='/js/lg-thumbnail.min.js'></script>
<script src='/js/lg-autoplay.min.js'></script>
<script src='/js/lg-zoom.min.js'></script>
<script src='/js/lg-video.min.js'></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
$(document).ready(gcu.postPageHandler);
</script>
</body></html>