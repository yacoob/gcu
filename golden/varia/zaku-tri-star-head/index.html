<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <link href='https://cdn.jsdelivr.net/npm/bootstrap3@3/dist/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='/photos/full/gcu-00847.jpg' rel='image_src'/>
  <meta property='og:image' content='https://gcu.tactical-grace.net/photos/full/gcu-00847.jpg'/>
  <meta name='twitter:card' content='summary_large_image'/>
  <meta name='twitter:image' content='https://gcu.tactical-grace.net/photos/full/gcu-00847.jpg'/>
  <meta name='description' content='tactical grace presents: Zaku II Head Black Tri-Star custom'/>
  <meta name='twitter:title' content='tactical grace presents: Zaku II Head Black Tri-Star custom'/>
  <link href='https://gcu.tactical-grace.net/varia/zaku-tri-star-head/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>Varia: Zaku II Head Black Tri-Star custom | tactical grace</title>
  <meta content='Varia: Zaku II Head Black Tri-Star custom | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/varia/zaku-tri-star-head/'>Varia: Zaku II Head Black Tri-Star custom</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a data-toggle='tooltip' title='main page' href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a data-toggle='tooltip' title='contact the author' href='https://goo.gl/forms/iJEsfLjBy1FE86aQ2'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-envelope'></span></button></a>
    <a data-toggle='tooltip' title='grade page' href='/varia/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a data-toggle='tooltip' title='previous kit' href='/varia/wonfes-winter-2017/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2016-12-26'>2016-12-26</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00835.jpg' data-sub-html='Looks innocent enough.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00835.jpg' alt='Looks innocent enough.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00836.jpg' data-sub-html='The pistons at the bottom just love to fall apart.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00836.jpg' alt='The pistons at the bottom just love to fall apart.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00837.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00837.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00838.jpg' data-sub-html='None of that is visible upon completion :3'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00838.jpg' alt='None of that is visible upon completion :3' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00839.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00839.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00840.jpg' data-sub-html='Welcome to nubmark hell.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00840.jpg' alt='Welcome to nubmark hell.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00841.jpg' data-sub-html='&#34;Gahh!&#34;'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00841.jpg' alt='&#34;Gahh!&#34;' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00842.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00842.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00843.jpg' data-sub-html='One of the pieces that required glue - the main visor wiper.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00843.jpg' alt='One of the pieces that required glue - the main visor wiper.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00844.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00844.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00845.jpg' data-sub-html='Glue was required for all small protruding bits.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00845.jpg' alt='Glue was required for all small protruding bits.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00846.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00846.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00847.jpg' data-sub-html='Screwed up the decals a bit, had to improvise.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00847.jpg' alt='Screwed up the decals a bit, had to improvise.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00848.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00848.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00856.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00856.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
</div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='https://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/css/lightgallery.min.css" />
<script src="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/js/lightgallery.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-thumbnail.js@1/dist/lg-thumbnail.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-zoom.js@1/dist/lg-zoom.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-video.js@1/dist/lg-video.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-fullscreen.js@1/dist/lg-fullscreen.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-autoplay.js@1/dist/lg-autoplay.min.js"></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
document.addEventListener('DOMContentLoaded', gcu.postPageHandler);
</script>
</body></html>